Tests/Packet++Test/Tests/TcpTests.cpp

```diff
@@ -304,3 +308,30 @@ PTF_TEST_CASE(TcpChecksumInvalidRead)
 
```

Comment:
the test above should fail on little endian after your change - it will need another htons.

Reply:
No the CI passed correctly